cartridges/app_storefront_training/cartridge/controllers/Carousels.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var server = require('server');
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+var userLoggedIn = require('*/cartridge/scripts/middleware/userLoggedIn');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+
+server.get('Show',
+    server.middleware.https,
+    function(req, res, next) {
+    res.render('carousels/carousels');
+    next();
+})
+
+module.exports = server.exports();
```

Comment:
Always place a line break at the end of the file.

Reply:
Solved